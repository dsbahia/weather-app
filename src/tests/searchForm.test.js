import React from "react";
import { render } from "@testing-library/react";
import SearchForm from "../components/searchForm";

describe("SearchForm", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<SearchForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
