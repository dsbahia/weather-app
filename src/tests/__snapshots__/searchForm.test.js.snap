// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="search-form"
  >
    <input
      class="search-form"
      type="text"
      value=""
    />
    <button
      class="search-form"
      type="submit"
    >
      Search
    </button>
  </div>
</DocumentFragment>
`;
